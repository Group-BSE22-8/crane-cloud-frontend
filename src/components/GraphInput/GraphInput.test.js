/* eslint-disable no-undef */
import React from "react";
import { shallow } from "enzyme";
import GraphInput from ".";
const GraphInputProps = {
    formatString: jest.fn(),
    createTimestamp: jest.fn(),
    getDate: jest.fn(),
    trimMonthName: jest.fn(),
};
describe('Graph input component', () => {
  const GraphInputWrapper = shallow(<GraphInput {...GraphInputProps} />);
  it("graph input  should match snapshot", () => {
    expect(GraphInputWrapper).toMatchSnapshot();
  });
  it("should match the snapshot", () => {
    GraphInputWrapper.setProps(GraphInputProps);
    expect(GraphInputWrapper).toBeDefined();
    expect(GraphInputWrapper).toHaveLength(1);
  });
});
