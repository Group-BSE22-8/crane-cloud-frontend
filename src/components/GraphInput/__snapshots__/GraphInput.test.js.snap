// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Graph input component graph input  should match snapshot 1`] = `ShallowWrapper {}`;
